### PR TITLE
Query and update animals table

### PR DESCRIPTION
**Pull request by willy836**
**Milestones**

- [ ] Add a new column called species of type string
- [ ] Insert new animals into the animals table
- [ ] Make changes inside transactions
- [ ] Write queries based on animals table

Attached below are the screenshots:

Screenshot (124)

Screenshot (120)

Screenshot (121)

Screenshot (122)

Screenshot (125)

image

image

image

image

image

image

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -23,3 +23,40 @@ WHERE name NOT LIKE 'Gabumon';
 
 SELECT * FROM animals
 WHERE weight_kg >= 10.40 AND weight_kg <= 17.30;
+
+/* Begin of a transaction */
+BEGIN;
+
```
Comment (Owusu-Desmond) [inline queries.sql]: - [ ] Please from the project requirement, Inside a transaction update the `animals` table by setting the `species` column to `unspecified`. Verify that the change was made. Then roll back the change and verify that the `species` columns went back to the state before the transaction. 
Please make sure you do that and **Take a screenshot of the results of your actions.** :pray:

Comment (Owusu-Desmond) [inline queries.sql]: - [ ] Please from the project requirement, 
  - Inside a transaction:
    - Update the `animals` table by setting the `species` column to `digimon` for all animals that have a name ending in `mon`.
    - Update the `animals` table by setting the `species` column to `pokemon` for all animals that don't have `species` already set.
    - Commit the transaction.
    - Verify that change was made and persists after commit.

Please make sure you do that and **Take a screenshot of the results of your actions.** :pray:

Comment (Owusu-Desmond) [inline queries.sql]: - [ ] Please from the project requirement, 
  - Inside a transaction:
    - Delete all animals born after Jan 1st, 2022.
    - Create a savepoint for the transaction.
    - Update all animals' weight to be their weight multiplied by -1.
    - Rollback to the savepoint
    - Update all animals' weights that are negative to be their weight multiplied by -1.
    - Commit transaction

Please make sure you do that and **Take a screenshot of the results of your actions.** :pray:

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @willy836 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by willy836** [comment]
REQUIRED SCREENSHOTS:
Screenshot (126)

Screenshot (127)

Screenshot (129)

Screenshot (130)

Screenshot (131)

Screenshot (133)
Screenshot (134)
Screenshot (136)

Screenshot (137)
Screenshot (138)

File: queries.sql (hunk 1 above)
Comment (Gambit142) [reply queries.sql]: - [ ] The first reviewer asked that you start a transaction and update the `animals` table by setting animals whose names end in `mon` to _**Digimon**_ and the rest to **_Pokemon_**. Although your screenshot shows the results of such a transaction, the code is missing in your file. Kindly write the piece of code that sets the rest of the animals to _**Pokemon**_ and commit the transaction 👍 😎

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @willy836 👋, 

Great work on making the changes requested by a previous reviewer 👏 

## Highlights
- Everything that creates or changes the structure of the tables is in the _**schema.sql**_ file
- Everything that populates tables with data is in the _**data.sql**_ file
- Everything that is a query that helps to analyze data is in the _**queries.sql**_ file

You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

 ## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please, remember to tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by willy836** [comment]
Added the piece of code that sets the rest of the animals to Pokemon and committed the transaction.

**Review by hiromataba — APPROVED** [review]
# Status: Approved :tada: :tada: :tada: 
Hi @willy836     :wave: :wave: 

200w  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**